From: Gordon Ball <gordon@chronitis.net>
Date: Thu, 30 Jan 2020 12:59:47 +0000
Subject: Avoid manipulating PYTHONPATH in tests

Tests are now generating errors suggesting shadowing of the stdlib, and
it feels like this combined with relative environment set by pybuild
might be an explanation.

Also avoid setting JUPYTER_NO_CONFIG, which currently _appears_ to
conflict with the path logic being tested in eg, test_serverextension
---
 notebook/tests/launchnotebook.py | 2 --
 1 file changed, 2 deletions(-)

diff --git a/notebook/tests/launchnotebook.py b/notebook/tests/launchnotebook.py
index 426ce43..a0847d2 100644
--- a/notebook/tests/launchnotebook.py
+++ b/notebook/tests/launchnotebook.py
@@ -98,9 +98,7 @@ class NotebookTestBase(TestCase):
     def get_patch_env(cls):
         return {
             'HOME': cls.home_dir,
-            'PYTHONPATH': os.pathsep.join(sys.path),
             'IPYTHONDIR': pjoin(cls.home_dir, '.ipython'),
-            'JUPYTER_NO_CONFIG': '1', # needed in the future
             'JUPYTER_CONFIG_DIR' : cls.config_dir,
             'JUPYTER_DATA_DIR' : cls.data_dir,
             'JUPYTER_RUNTIME_DIR': cls.runtime_dir,
